doctest/doctest.h

```diff
@@ -454,6 +454,36 @@ namespace doctest {
 
 DOCTEST_INTERFACE extern bool is_running_in_test;
 
+#if _MSC_VER
```

Comment:
This breaks on other compilers when it's undefined - all builds besides windows clang/cl fail because of this - maybe use DOCTEST_MSVC? Builds must pass before this can get merged.

Also, it would be good to have a good fat comment explaining why this code is needed so that people don't have to look at git history & git blame.